tests/backbeat/tests/api/metrics.js

```diff
@@ -27,49 +31,90 @@ function makeDelayedGETRequest(endpoint, cb) {
     }, 5000);
 }
 
-describe.skip('Backbeat replication metrics', function dF() {
+describe('Backbeat replication metrics', function dF() {
     this.timeout(REPLICATION_TIMEOUT);
     const roleArn = 'arn:aws:iam::root:role/s3-replication-role';
+    const storageClass = `${destAWSLocation},${destAzureLocation}`;
+    const pathPrefix = '/_/backbeat/api/metrics/crr';
 
     beforeEach(done => series([
         next => scalityUtils.createVersionedBucket(srcBucket, next),
         next => scalityUtils.putBucketReplicationMultipleBackend(srcBucket,
-            destBucket, roleArn, destLocation, next),
+            'placeholder', roleArn, storageClass, next),
     ], done));
 
-    afterEach(done => series([
+    afterEach(done => parallel([
```

Comment:
Looks like it could be cleaner to use two describe statements because these `beforeEach` and `afterEach` hooks are not needed for the tests that are checking the API route properties (e.g. the test `should return all metrics for path ${path}`). That is, those tests should pass without having created a bucket and setup replication on it.